app/src/org/commcare/android/view/SquareButtonWithText.java

```diff
@@ -110,10 +130,4 @@ public void setTextColor(int textColor) {
     }
 
     //endregion
-
-    @Override
-    public void setOnClickListener(OnClickListener l) {
-        // attach the listener to the squareButton instead of the layout
-        squareButton.setOnClickListener(l);
-    }
```

Comment:
This prevents the full widget from toggling its selector unless you click on the area that is outside of the square button, since the selector (the thing that makes the button change color) is triggered by the signals from the onclicklistener.